### PR TITLE
Add 1.23.2, hotfixes to prev 1.23 versions, and update docker version in test data

### PR DESCRIPTION
**Pull request by ganeshkumar5699**

### CODE REVIEW

File: pkg/agent/testdata/AKSWindows2019+CustomCloud/CustomData [hunk 1]
```diff
@@ -89,7 +89,7 @@ $global:ContainerdUrl = "https://k8swin.blob.core.windows.net/k8s-windows/contai
 $global:ContainerdSdnPluginUrl = ""
 
 ## Docker Version
-$global:DockerVersion = "20.10.6"
+$global:DockerVersion = "20.10.9"
```
Comment (alexeldeib) [inline pkg/agent/testdata/AKSWindows2019+CustomCloud/CustomData:92]: where'd this come from? missed make generate somewhere?

**Review by alexeldeib — APPROVED** [review]
lgtm but curious how that slipped in

File: pkg/agent/testdata/AKSWindows2019+CustomCloud/CustomData (hunk 1 above)
Comment (ganeshkumar5699) [reply pkg/agent/testdata/AKSWindows2019+CustomCloud/CustomData:92]: yes, I think `make generate` was not run with this commit: https://github.com/Azure/AgentBaker/commit/0d64f6cd36ae463ef383ca42b007ec6ad772babd

Comment (ganeshkumar5699) [reply pkg/agent/testdata/AKSWindows2019+CustomCloud/CustomData:92]: Perhaps, the CI doesn't check that `make generate` updates files in `testdata`. Should check and fix that in the future, in that case.

**Pull request merged**